Cargo.toml

```diff
@@ -15,3 +15,4 @@ rrg-proto = { path = "proto/" }
 simplelog = { version = "0.7.6" }
 structopt = { version = "0.3.12" }
 sys-info = { version = "0.5.1" }
```

Comment:
Recently, I have made a change to use the `sysinfo` crate rather than `sys-info`. `sys-info` does not seem to work on other operating systems and is more low-level, whereas `sysinfo` gives you more abstractions and works on pretty much all supported platforms and saves us from having a lot of `unsafe` calls. Have you considered using it?

Right now, the code will not compile on Windows and macOS, I believe.

Reply:
I try to find something useful for my action in crate `sysinfo`, but, seems to me, this package doesn't provide any information about OS, only about system at all.
Also as documentation of [sys-info](https://docs.rs/sys-info/0.7.0/sys_info/) says that it supports Windows and Mac OS too.